README.rst

```diff
@@ -55,3 +61,10 @@ Style guide
 This project uses the `Django coding style guide`_.
 
 .. _Django coding style guide: https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/
+
+Documentation
+^^^^^^^^^^^^^
+
+To create the docs locally, go to the ``docs`` directory, install the requirements, and run ``make html``.
+
+The resulting ``index.html`` file will be in ``_build/html/``.
```

Comment:
Might be nice to display this as a list of bash commands from the repo root (here and in the contributing docs)

Reply:
The contributing docs do show some of the commands; I was just thinking of something more like:

Text description of commands.

```
$ command 1
$ command 2
$ command 3
```

of course the disadvantage of providing exact commands is they won't work on all platforms :-p